test/Vagrantfile

```diff
@@ -9,13 +9,22 @@ $K8S_NODES = (ENV['K8S_NODES'] || "2").to_i
 $NFS = ENV['NFS']=="1"? true : false
 $SERVER_BOX = (ENV['SERVER_BOX'] || "cilium/ubuntu-dev")
 $SERVER_VERSION="120"
+$SERVER_VERSION= "119"
```

Comment:
Why are there two "SEVER_VERSION" values here? Can you squash this with 7a2597bf41aabdbe4ed48b2391e4fe43a8ad7985 in case we need to bisect?